recourse_methods/mrmc_method.py

```diff
@@ -92,47 +92,85 @@ def alpha(dist: np.ndarray) -> np.ndarray:
 
 
 class MRM:
-    """Monotone Recourse Measures generates recourse directions."""
+    """Monotone Recourse Measures (MRM) generates recourse directions."""
 
     def __init__(
         self,
         dataset: pd.DataFrame,
         adapter: recourse_adapter.RecourseAdapter,
         alpha: AlphaFunction = get_volcano_alpha(),
-        rescale_direction: RecourseRescaler = normalize_rescaler,
+        rescale_direction: Optional[RecourseRescaler] = normalize_rescaler,
+        confidence_threshold: Optional[float] = None,
+        model: Optional[model_interface.Model] = None,
+        _processed_data: Optional[recourse_adapter.EmbeddedDataFrame] = None,
     ):
         """Creates an MRM instance.
 
+        The dataset is processed for recourse using MRM.process_data(). This
+        processing step can be skipped by providing recourse data directly via
+        _processed_data. This is typically unnecessary for end users.
```

Comment:
Nit: describing the function `MRM.process_data()` and private class attribute `_processed_data` is a little confusing. I'd suggest changing the 2nd sentence to:

"The data processing step can be skipped by directly assigning the class attribute `_processed_data` with data ready for recourse."

or something like this.